charts/hello/templates/deployment.yaml

```diff
@@ -13,6 +13,12 @@ spec:
     metadata:
       labels:
         {{- include "hello.selectorLabels" . | nindent 8 }}
+      {{- if .Values.podAnnotations }}
```

Comment:
Please use "with" instead of "if" so you can refer to the "." inside. More DRY

Reply:
ok, changed